google/devtools/cloudtrace/v1/trace_gapic.yaml

```diff
@@ -0,0 +1,68 @@
+type: io.gapi.vgen.ConfigProto
+language_settings:
+  python:
+    package_name: google.devtools.cloudtrace.v1
+  ruby:
+    package_name: Google::Devtools::Cloudtrace::V1
+interfaces:
+- name: google.devtools.cloudtrace.v1.TraceService
+  collections:
+  - name_pattern: projects/{project}/traces/{trace_id}
```

Comment:
Having chatted to the cloud trace team, you can remove this as it's misleading.
In the trace api, the ids are just the raw ids, and not resource names